src/StyleRuntime.js

```diff
@@ -0,0 +1,58 @@
+// @flow
```

Comment:
If you are moving styles to a plugin, then these definitions should be part of the plugin, not licit itself.